pkg/cmd/server/api/validation/oauth.go

```diff
@@ -87,6 +88,8 @@ func ValidateOAuthConfig(config *api.OAuthConfig) ValidationResults {
 			)))
 	}
 
+	validationResults.AddErrors(login.ValidateLoginTemplate(config.Templates)...)
```

Comment:
do the `if config.Templates != nil { }` bits here, as well as generic file existence validation, then read the content and pass the content to `ValidateLoginTemplate`